public/app/plugins/datasource/loki/components/LokiQueryEditor.tsx

```diff
@@ -29,17 +29,16 @@ export const testIds = {
 };
 
 export const LokiQueryEditor = React.memo<LokiQueryEditorProps>((props) => {
-  const { onChange, onRunQuery, onAddQuery, data, app, queries, datasource } = props;
+  const { onChange, onRunQuery, onAddQuery, data, app, queries, datasource, range } = props;
```

Comment:
```suggestion
  const { onChange, onRunQuery, onAddQuery, data, app, queries, datasource, range: timeRange } = props;
```

And you can keep the same name it used to have, which I think is a better fit for it.

Reply:
Good idea! Fixed in https://github.com/grafana/grafana/pull/75106/commits/443764046c509ba1c399274525955d2e38ed6a8a